boxsdk/object/item.py

```diff
@@ -398,3 +399,133 @@ def remove_from_collection(self, collection):
             'collections': updated_collections
         }
         return self.update_info(data)
+
+    def collaborate(self, accessible_by, role, can_view_path=None, notify=None, fields=None):
+        """Collaborate user or group onto a Box item.
+
+        :param accessible_by:
+            An object containing the collaborator.
+        :type accessible_by:
+            class:`User` or class:`Group`
+        :param role:
+            The permission level to grant the collaborator.
+        :type role:
+            `unicode`
+        :param can_view_path:
+            Indicates whether the user can view the path of the folder collaborated into.
+        :type can_view_path:
+            `bool` or None
+        :param notify:
+            Determines if the collaborator should receive a notification for the collaboration.
+        :type notify:
+            `bool` or None
+        :param fields:
+            List of fields to request.
+        :type fields:
+            `Iterable` of `unicode`
+        """
+        url = self._session.get_url('collaborations')
+        body = {
+            'item': {
+                'type': self.object_type,
+                'id': self.object_id,
+            },
+            'accessible_by': {
+                'type': accessible_by.object_type,
+                'id': accessible_by.object_id,
+            },
+            'role': role,
+        }
+        if can_view_path is not None:
+            body['can_view_path'] = can_view_path
+        params = {}
+        if fields is not None:
+            params['fields'] = ','.join(fields)
+        if notify is not None:
+            params['notify'] = ','.join(fields)
```

Comment:
This looks like an incorrect copy/paste; should probably be just `params['notify'] = notify`